app/src/main/java/com/team7/cmput301/android/theirisproject/model/Problem.java

```diff
@@ -4,16 +4,20 @@
 
 package com.team7.cmput301.android.theirisproject.model;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
 import java.util.ArrayList;
 import java.util.Date;
 import java.util.List;
+import java.util.Locale;
 
 import io.searchbox.annotations.JestId;
 
 public class Problem {
     @JestId
     private String _id;
 
+    transient private SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss", Locale.CANADA);
```

Comment:
We may be using this for records,comments so I think you can put this in IrisProjectApplication as a singleton

Reply:
You can make a DateHelper class in the helper package I added and put this as a variable there